attributes/default.rb

```diff
@@ -12,22 +12,18 @@
 default['kafka-cluster']['config']['path'] = '/etc/kafka/kafka.properties'
 default['kafka-cluster']['config']['properties']['broker.id'] = 1
 default['kafka-cluster']['config']['properties']['port'] = 9_092
-default['kafka-cluster']['config']['properties']['num.partitions'] = 8
```

Comment:
Are these values no longer needed as per latest version?

Reply:
So the metrics are probably not what most people will want out of the box as it wants to write csv files (there are other ways to get kafka metrics). This makes it optional. The other pieces are mostly me sorting the keys.